### DISCUSSION TITLE
Ragdolls Not Colliding with Each Other

### DISCUSSION THREAD

**Discussion by okamototomoyuki** [jrouwe/JoltPhysics.js#214] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/73733598-c5ea-4591-aeaf-ffaea25c44c2)
Hello, I am currently using ragdolls. I’ve encountered an issue where two ragdolls on the same layer pass through each other without colliding. However, both ragdolls collide properly with the ground, which is also on the same layer.  
Could you please advise on how I might get the ragdolls to collide with each other as well?  
Thank you very much for your assistance.

**Comment by jrouwe** [comment, marked as answer]
You do not say what kind of filtering you are using to prevent body parts from the same ragdoll from colliding. I'm going to assume you're using `GroupFilterTable`, possibly through calling `RagdollSettings::DisableParentChildCollisions`. In this case you have to call `Ragdoll::SetGroupID` on each ragdoll and give a unique ID to each of them so they will collide.

**Reply by okamototomoyuki** [reply]
Thank you for your guidance!  
As you assumed, I was indeed using `RagdollSettings::DisableParentChildCollisions`.  
After setting different IDs for each ragdoll using `Ragdoll::SetGroupID`, they now collide with each other as expected.  
I appreciate your help and quick response!
